megameklab/src/megameklab/ui/MenuBar.java

```diff
@@ -410,7 +413,10 @@ private JMenu createHTMLUnitExportMenu() {
         final JMenuItem miExportCurrentUnitToHTML = new JMenuItem(resources.getString("CurrentUnit.text"));
         miExportCurrentUnitToHTML.setName("miExportCurrentUnitToHTML");
         miExportCurrentUnitToHTML.setMnemonic(KeyEvent.VK_U);
-        miExportCurrentUnitToHTML.addActionListener(evt -> exportSummary(true));
+        miExportCurrentUnitToHTML.addActionListener(evt -> {
+            warnOnInvalid();
+            exportSummary(true);
```

Comment:
Is it warning twice here with the warning in the called method?

Reply:
Sure is, didn't even notice earlier. I'll get that corrected.